app/routes.py

```diff
@@ -1,2 +1,77 @@
-from flask import Blueprint
+from crypt import methods
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, abort, make_response, request
 
+planet_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+def validate_id(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response({"message": f"{cls.__name__} {model_id} is invalid"}, 400))
+
+    planet = cls.query.get(model_id)
```

Comment:
Since you've refactored this method to be generalizable to all models, you'd want to update the variable name here from planet to something more generic like `model`